src/etc/htmldocck.py

```diff
@@ -483,11 +537,19 @@ def check(target, commands):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 3:
-        stderr('Usage: {} <doc dir> <template>'.format(sys.argv[0]))
+    if len(sys.argv) not in [3, 4]:
+        stderr('Usage: {} <doc dir> <template> [--bless]'.format(sys.argv[0]))
         raise SystemExit(1)
 
-    check(sys.argv[1], get_commands(sys.argv[2]))
+    rust_test_path = sys.argv[2]
+    if len(sys.argv) > 3 and sys.argv[3] == '--bless':
+        bless = True
+    else:
+        # FIXME: for now, we only support `--bless` at the end.
```

Comment:
I don't think this needs to be a FIXME; only x.py usually calls htmldocck so it's not a big deal if the arguments are inflexible.

Reply:
Fixed.